### PR TITLE
[AU4] Show volume tooltip immediately on drag

### PR DESCRIPTION
**Pull request by kryksyh**
Resolves: #7522 

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml [hunk 1]
```diff
@@ -62,8 +62,13 @@ StyledPopupView {
         }
     }
 
-    function show() {
-        openTimer.restart()
+    function show(noDelay = false) {
```
Comment (grliszas14) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml:65]: Just minor nitpick about formatting to match overall qml style we have:
`        if (noDelay) {
            open();
        } else {
            openTimer.restart()
        }`

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml [hunk 2]
```diff
@@ -62,8 +62,13 @@ StyledPopupView {
...
+        if(noDelay) {
+            open();
+        }
+        else {
+            openTimer.restart()
+        }
```
Comment (grliszas14) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml:71]: Just a minor nitpick about formatting to match overall QML style we have:
```qml
        if (noDelay) {
            open();
        } else {
            openTimer.restart()
        }
```

**Review by grliszas14 — APPROVED** [review]

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml (hunk 2 above)
Comment (Eism) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml]: also space after `if` and without `;` :)

Comment (kryksyh) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/audio/VolumeTooltip.qml]: ohh, someone has to write a style checker for qml, i cant stop putting semicolons at the end of the line

**Review by Eism — APPROVED** [review]

**Pull request merged**